specs/binding-kafka.spec/src/test/java/io/aklivity/zilla/specs/binding/kafka/config/SchemaTest.java

```diff
@@ -117,7 +117,7 @@ public void shouldValidateClientWhenTopic()
     @Test
     public void shouldValidateCacheOptionsCatalog()
     {
-        JsonObject config = schema.validate("cache.options.type.avro.yaml");
+        JsonObject config = schema.validate("cache.options.type.test.yaml");
```

Comment:
Please rename config to `cache.options.validate.yaml`.

Reply:
done